mitodl/mitxonline
# Flexible pricing clean up email template

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/727

#### What's this PR do?
Adds formatting to the email sent to a learner when their flexible pricing request is reset through the refine staff-dashboard. 

#### How should this be manually tested?

1. Without this PR, with a flexible pricing request which has been previously submitted, login to the staff-dashboard (http://mitxonline.odl.local:8013/staff-dashboard/flexible_pricing) with an `mitxonline` staff user.  
2. Click the 'reset' button for the flexible pricing request you are using to test.
3. The email sent to that user should appear as one single paragraph, without any formatting.
4. Using the functionality integrated with this PR, complete step 2 again.
5. The email sent to that user should appear with formatting as shown below.


#### Screenshots
![Screen Shot 2022-07-26 at 2 04 43 PM](https://user-images.githubusercontent.com/8311573/181090650-b73cb8cf-3648-4294-9e10-6b355f788238.png)

The screenshot I have provided uses the logo from production.
